.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -37,11 +39,14 @@ body:
       Ensure that the screenshots/videos are full screen and do not contain any sensitive information such as API keys or Access Tokens. 
   validations:
     required: false
-- type: input
+- type: dropdown
   attributes:
     label: Operating System
     description: Which OS are you using?
-    placeholder: "Windows/macOS/Linux"
+    options:
+      - Windows
+      - macOS
+      - Linux
```

Comment:
What would _Other_ represent in this context?

Reply:
Unsure! Since it was free form text there's really no way for me to know. But I do think these are obviously the major OSs and the only ones we support, so I think this is probably fine as-is.